internal/cmd/cmd.go

```diff
@@ -24,7 +24,7 @@ func Make() *cobra.Command {
 	app.AddCommand(
 		loginCmd(),
 		logoutCmd(),
-		shCmd(),
```

Comment:
We cannot remove this command because SSH is optional.

Reply:
I'm working on some docs and supplementary material, but I basically think that is a bad requirement and doesn't make sense anymore.

Reply:
SSH being optional will probably need to be deprecated and not removed outright since it's a major change. Keeping the old shell command will be necessary until SSH is required.